deployment-local/pom.xml

```diff
@@ -0,0 +1,173 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <parent>
+        <groupId>org.wso2.testgrid</groupId>
+        <artifactId>testgrid-parent</artifactId>
+        <version>1.0.8-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>org.wso2.testgrid.deployment.local</artifactId>
+    <name>WSO2 TestGrid Project - Deployment Local POM</name>
```

Comment:
When I unzip the distribution, I saw multiple different files directly under root folder location. This is quite confusing for a new user. Hence, please move files users does not interact with to a separate folder.